wlhosted/templates/legal/documents/tos.html

```diff
@@ -82,44 +82,44 @@ <h2>{% blocktrans %}Liability for Damage{% endblocktrans %}</h2>
+<p class="item">{% blocktrans %}Within the meaning of Article 5 (2) of Act No. {{ law_101_2000 }} Coll., on personal data protection, as amended, the User hereby gives consent to the collecting, storage and processing of the Personal Data provided by the User to the administrator, who is the Provider, through the use of the Service; the User gives consent to the processing and use of the User’s Personal Data by the Provider, the Providers employees and/or the Providers authorised business partners. This consent is given as long as User does not remove their account from the Service.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}The purpose of the processing of Personal Data as determined by the Provider is the use of Personal Data for sending of commercial communications to the subject of the data (the User) through electronic means under Act No. {{ law_480_2004 }} Coll., on certain services of the information society, as amended, and further for marketing and business needs of the Provider’s contractual partners and for statistical and/or marketing purposes of the Provider.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The purpose of the processing of Personal Data as determined by the Provider is the use of Personal Data for sending commercial communications to the subject of the data (the User) through electronic means under Act No. {{ law_480_2004 }} Coll., on certain services of the information society, as amended, and further for marketing and business needs of the Provider’s contractual partners and for statistical and/or marketing purposes of the Provider.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}If the User directly and effectively informs the Provider that he no longer wishes to receive business information from the Provider, the Provider shall refrain from sending further unsolicited business communications.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}If the User directly and effectively informs the Provider of no longer wishing to receive business information from the Provider, the Provider shall refrain from sending further unsolicited business communications.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}The User declares that he/she is aware of his/her rights under Article 12 and Article 21 of the Personal Data Protection Act and declares that all the Personal Data are accurate and true and have been provided voluntarily.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The User declares awareness of own rights under Article 12 and Article 21 of the Personal Data Protection Act and declares that all the Personal Data is accurate and true and has been provided voluntarily.{% endblocktrans %}</p>
```

Comment:
Following a debate with ssantos at hosted Weblate,
There could be links to 
http://www.privacy-regulation.eu/en/article-21-right-to-object-GDPR.htm  
and
http://www.privacy-regulation.eu/en/article-12-transparent-information-communication-and-modalities-for-the-exercise-of-the-rights-of-the-data-subject-GDPR.htm
I don't know that these are the best ones, but a start.